From 4177e8935464a552aefa21439e4549df1d330ac8 Mon Sep 17 00:00:00 2001
From: Amir Ayupov <aaupov@fb.com>
Date: Wed, 1 Feb 2023 12:35:46 -0800
Subject: [PATCH] [BOLT][CMake] Add dependency on llvm_vcsrevision_h

The dependence is needed since Utils includes VCSRevision.h, and other
LLVM components that include this header also have the llvm_vcsrevision_h
dependency.

Fixes #60460.

Reviewed By: #bolt, ayermolo

Differential Revision: https://reviews.llvm.org/D143101
---
 bolt/lib/Utils/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/bolt/lib/Utils/CMakeLists.txt b/bolt/lib/Utils/CMakeLists.txt
index 9ed9be6e1d85ff..d1403314274bd6 100644
--- a/bolt/lib/Utils/CMakeLists.txt
+++ b/bolt/lib/Utils/CMakeLists.txt
@@ -7,6 +7,9 @@ add_llvm_library(LLVMBOLTUtils
   LINK_LIBS
   ${LLVM_PTHREAD_LIB}

+  DEPENDS
+  llvm_vcsrevision_h
+
   LINK_COMPONENTS
   Support
   )